### DISCUSSION TITLE
Subscribe Private

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67319] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I subscribe my private feed? I remember there was a "subscribe to your feed" on dashboard

**Comment by mehulrajdev** [comment, marked as answer]
Follow GitHub's main website and mostly make feed public

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
